messages/src/utils/error_codes.rs

```diff
@@ -67,44 +69,44 @@ pub fn map_indy_error_code<C: PrimInt>(error_code: C) -> u32 {
     }
 }
 
-#[cfg(test)]
-#[cfg(feature = "general_test")]
-pub mod unit_tests {
-    use vdrtoolsrs::ErrorCode;
-
-    use crate::utils::devsetup::SetupEmpty;
-
-    use super::*;
-
-    #[test]
-    fn test_invalid_param_err() {
-        let _setup = SetupEmpty::init();
-
-        let err100: IndyError = IndyError {
-            error_code: ErrorCode::CommonInvalidParam1,
-            message: String::new(),
-            indy_backtrace: None,
-        };
-        let err107: IndyError = IndyError {
-            error_code: ErrorCode::CommonInvalidParam8,
-            message: String::new(),
-            indy_backtrace: None,
-        };
-        let err111: IndyError = IndyError {
-            error_code: ErrorCode::CommonInvalidParam12,
-            message: String::new(),
-            indy_backtrace: None,
-        };
-        let err112: IndyError = IndyError {
-            error_code: ErrorCode::CommonInvalidState,
-            message: String::new(),
-            indy_backtrace: None,
-        };
-
-        assert_eq!(MessagesError::from(err100).kind(), MesssagesErrorKind::InvalidLibindyParam);
-        assert_eq!(MessagesError::from(err107).kind(), MesssagesErrorKind::InvalidLibindyParam);
-        assert_eq!(MessagesError::from(err111).kind(), MesssagesErrorKind::InvalidLibindyParam);
-        // Test that RC 112 falls out of the range 100...112
-        assert_ne!(MessagesError::from(err112).kind(), MesssagesErrorKind::InvalidLibindyParam);
-    }
-}
+// #[cfg(test)]
```

Comment:
I think this test was copy paster from `aries-vcx` in the first place, lets remove this

Reply:
And in general, we should probably extract out in future some `aries-vcx-error` crate and reuse it in `aries-vcx`, `messages`, `agency_client`